test/sanity/run-test.sh

```diff
@@ -50,4 +59,4 @@ _output/smbplugin --endpoint "$endpoint" --nodeid "$nodeid" -v=5 &
 
 echo 'Begin to run sanity test...'
 readonly CSI_SANITY_BIN='csi-sanity'
-"$CSI_SANITY_BIN" --ginkgo.v --ginkgo.noColor --csi.endpoint="$endpoint" --ginkgo.skip='should fail when the requested volume does not exist|should work|create a volume with already existing name and different capacity|should be idempotent|should fail when volume does not exist on the specified path'
+"$CSI_SANITY_BIN" --ginkgo.v --csi.testvolumeparameters="$(pwd)/test/sanity/params.yaml" --csi.endpoint="$endpoint" --ginkgo.skip='should fail when the requested volume does not exist|should work|create a volume with already existing name and different capacity|should be idempotent|should fail when volume does not exist on the specified path'
```

Comment:
remove `should fail when volume does not exist on the specified path` case, that's what https://github.com/kubernetes-csi/csi-driver-smb/issues/182 wants to do, thanks.

Reply:
Should I also change `share: /` to `share: /sambashare` in `params.yaml` for the tests to work?